docs/swarm/how-swarm-mode-works/services.md

```diff
@@ -0,0 +1,101 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/swarm/how-swarm-mode-works/"
```

Comment:
@thaJeztah , if possible, i think you should leave it on nodes.md. there's no index.md for this, so if someone manually navigates to "/engine/swarm/how-swarm-mode-works/" it directs them to nodes.

Should definitely be removed from services.md